kitsune/customercare/zendesk.py

```diff
@@ -38,22 +47,38 @@ def __init__(self, **kwargs):
         }
         self.client = Zenpy(**creds)
 
-    def _user_to_zendesk_user(self, user, include_email=True):
-        fxa_uid = user.profile.fxa_uid
-        id_str = user.profile.zendesk_id
+    def _user_to_zendesk_user(self, user, include_email=True, ticket_fields=None):
```

Comment:
do we still the include_email? With the loginless option, maybe we can drop this flag completely and we can also remove the `email` field from the `if/else` and replace it with:

email = ticket_fields.get("email") or user.email in the ZendeskUser

Alternatively we can always have `email = ticket_fields.get("email") and populate the ticket_fields either from the form or from the user object.

Reply:
Agreed - moving it to a single statement at the ZendeskUser call.